src/uu/ls/src/ls.rs

```diff
@@ -619,7 +619,7 @@ fn display_file_name(
     let mut width = UnicodeWidthStr::width(&*name);
 
     let color = match options.opt_str("color") {
-        None => true,
+        None => stdout_isatty(),
```

Comment:
isatty is deprecated, I moved ls to atty

```suggestion
        None => atty::is(atty::Stream::Stdout),
```